app.manifest

```diff
@@ -1,46 +1,46 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I'm just going to fix this here. It was committed with incorrect line endings and keeps coming up.